Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -821,7 +821,7 @@ func searchUsers(_ users: [ChatUser], by searchInput: String, excludingId: Strin
     let searchInput = normalize(searchInput)
 
     let matchingUsers = users.filter { $0.id != excludingId }
-        .filter { $0.id.contains(searchInput) || (normalize($0.name ?? "").contains(searchInput)) }
+        .filter { searchInput == "" || $0.id.contains(searchInput) || (normalize($0.name ?? "").contains(searchInput)) }
 
     let uniqueUsers = matchingUsers.reduce(into: [String: ChatUser]()) {
```

Comment:
How not `Set(matchingUsers)` ? And then `Array(uniqueUsers).sorted...`, The reduce seems a bit unnecessary I guess?

Reply:
good point, ChatUser is indeed hashable :)